src/datepicker/stories/DateRangePicker.stories.tsx

```diff
@@ -1,21 +1,51 @@
-import "./index.css";
 import * as React from "react";
 import { format } from "date-fns";
 import { Meta, Story } from "@storybook/react";
 import { useArgs } from "@storybook/client-api";
 import { addWeeks, setDate, subWeeks } from "date-fns";
+import { DEFAULT_REACT_CODESANDBOX } from "storybook-addon-preview";
 
-import DateRangePickerComponent from "./DateRangePickerComponent";
+import {
+  rangeAppTemplate,
+  rangeAppTemplateJs,
+  rangeCssTemplate,
+} from "./templates";
+import "./DatePicker.css";
+import { App as DateRangePicker } from "./DateRangePicker.component";
 
 export default {
   title: "DateRangePicker",
-  component: DateRangePickerComponent,
+  component: DateRangePicker,
   argTypes: {
     start: { control: "date", name: "value.start" },
     end: { control: "date", name: "value.end" },
     minValue: { control: "date" },
     maxValue: { control: "date" },
   },
+  parameters: {
+    preview: [
+      {
+        tab: "ReactJS",
+        template: rangeAppTemplateJs,
+        language: "jsx",
+        copy: true,
+        codesandbox: DEFAULT_REACT_CODESANDBOX(["renderless-components@alpha"]),
+      },
+      {
+        tab: "React",
+        template: rangeAppTemplate,
+        language: "tsx",
+        copy: true,
+        codesandbox: DEFAULT_REACT_CODESANDBOX(["renderless-components@alpha"]),
+      },
+      {
+        tab: "CSS",
+        template: rangeCssTemplate,
+        language: "css",
+        copy: true,
+      },
+    ],
```

Comment:
Can we create a helper function like this? 
![image](https://user-images.githubusercontent.com/35374649/97987127-c93d4080-1e00-11eb-9e90-3b2dbaa4c477.png)


I think it would be nicer.

Reply:
Yeah, we can do that !!